src/components/WeatherWidgetCurrentData.vue

```diff
@@ -0,0 +1,88 @@
+<template>
+  <section class="weather-info">
+    <div class="weather-info__image-wrapper">
+      <img
+        class="weather-info__image"
+        :src="props.weatherIcon"
+        :alt="`${props.weatherDescription} Weather Image`"
+      />
+      <p class="weather-info__description">
+        {{ props.weatherDescription }}
+      </p>
+    </div>
+    <div class="weather-info__temperature-wrapper">
+      <h3 class="weather-info__temperature-info">
+        {{ props.temperature }}&deg;C
+      </h3>
+    </div>
+    <ul class="weather-info__additional-info">
+      <li class="weather-info__additional-info-item">
+        Wind: {{ props.windSpeed }}
+      </li>
+      <li class="weather-info__additional-info-item">
+        Pressure: {{ props.pressure }}
+      </li>
+      <li class="weather-info__additional-info-item">
+        UV Index: {{ props.uvIndex }}
+      </li>
+    </ul>
+  </section>
+</template>
+
+<script setup lang="ts">
+export type WeatherData = {
+  temperature: number;
+  weatherIcon: string;
+  weatherDescription: string;
+  windSpeed: number;
+  pressure: number;
+  uvIndex: number;
+  date?: string;
+};
+
+const props = defineProps<WeatherData>();
+</script>
+
+<style scoped lang="scss">
+.weather-info {
+  grid-template-columns: repeat(3, 1fr);
+  display: grid;
```

Comment:
```suggestion
  display: grid;
  grid-template-columns: repeat(3, 1fr);
```
display: grid; should go before other grid properties for better visibility,